### ISSUE TITLE
Apostrophe 2: Invisible text when commenting

### ISSUE THREAD

**Issue by markx2** [Automattic/themes#279]
## Steps to replicate
Open https://themusing.in on mobile (I used Firefox on Android)
Open a post, scroll down and start commenting.

## Result

You cannot see your text as you type.

You can only see it when you move to a different field.

## Expected

That I could see my comment as I typed

* User report: 7179510-hc
* Follow-up ticket: Needs doing
* User's site: See above

**Comment by laurelfulford** [comment]
I haven't been able to recreate this issue -- I think it must be specific a certain browser/OS version. I tried on my iPhone 6, plus the following combos in Browserstack:

Galaxy S5, Firefox
Galaxy S8, Firefox
Pixel 3, Firefox

The page I tried commenting on was: https://themusing.in/2018/11/22/united-nations-security-council/

I also tried one on my own test site, since it looks like the above site has custom colours, and I'm not sure it always did -- they could be hiding the issue. 

@markx2 is there a specific device model/Android version or Firefox version that you recreated this with? Any additional info would be a huge help - thanks!
